LICENSE.txt

```diff
@@ -12,14 +12,19 @@ licenses).  It is your responsibility to comply with the applicable licenses.
 Information on third party materials, and the applicable license terms, are 
 referenced in or included with the materials, such as in separate LICENSE.txt 
 files accompanying the materials.
+
+Third party components with license-identification SPDX tags are logged in 
+the 3P repository at https://github.com/o3de/3p-package-source. For more 
+information on applying SPDX tags to O3DE components properly, see 
+https://www.o3de.org/docs/user-guide/build/packages/#o3de-package-structure.
  
 Please note that certain materials are subject to "copyleft" licenses, which 
 require distribution of source code, including:
  
 - Qt Toolkit https://github.com/qtproject/, which is subject to the GNU 
 Lesser General Public License version 3 (with certain exceptions). A copy of 
 the source code for Qt Toolkit may be found at 
-https://s3-us-west-2.amazonaws.com/ly-legal/LicenseConformance/Qt/Src.zip
+https://github.com/o3de/3p-package-source/tree/main/package-system/Qt
```

Comment:
Just wanted to point out this link is actually just to our 3p scripts we use to build Qt, not the Qt source itself. Based on the line above it (`A copy of the source code for Qt Toolkit may be found at`) should this link be to the actual source?

If so, the link to our fork of Qt and the branch we build from is here:
https://github.com/o3de/qt5/tree/5.15.1-o3de

Reply:
Thanks for the clarification. It's fixed